roles/cloud_connector/templates/rhc-worker-cloud-connector.toml.j2

```diff
@@ -0,0 +1,5 @@
+---
+foreman: "{{ foreman_url }}"
+token: "{{ foreman_access_token }}"
+username: "{{ foreman_username }}"
+require_ssl: "{{ foreman_require_ssl }}"
```

Comment:
According to the PR in the worker, we are not using a separate toml file. This template won't be used

Reply:
Yea this whole thing is old and needs to be dropped from the role.